v2/cf-nodejs-logging-support/src/lib/config/config.ts

```diff
@@ -129,6 +132,18 @@ export default class Config {
             }
 
             file.fields?.forEach(field => {
+                if (field.envVarRedact || field.envVarSwitch) {
+                    const shouldBeReduced = this.isReducedField(field);
+                    if (shouldBeReduced) {
+                        if (field.envVarRedact) {
+                            this.redactedFields.set(field.name, true);
+                        } else {
+                            this.omittedFields.set(field.name, true);
+
+                        }
+                    }
+                }
```

Comment:
This implementation works fine for fields with envVarRedact. However, it does not consider the environment variable set for envVarSwitch. E.g.:

* Field X has envVarSwitch, the corresponding env var is set to true => X omitted ⚡️

Also consider that fields can have envVarRedact and envVarSwitch set to the same or different environment variables.